lib/canvas.js

```diff
@@ -111,3 +130,5 @@ Canvas.prototype.toDataURL = function (a1, a2, a3) {
     return `data:${type};base64,${this.toBuffer(type, opts).toString('base64')}`
   }
 }
+
+module.exports = Canvas
```

Comment:
Can you add a final newline here?

```suggestion
module.exports = Canvas
```